### ISSUE TITLE
Collapsed Image is not formatted

### ISSUE THREAD

**Issue by TheFabulousPika** [TheFabulousPika/steam-chatlog-to-text#36]
```
<div class="msgText " data-copytext="">
  <div class="BBCodeAlreadyCollapsed Minimized">
    <div class="BBCodeAlreadyCollapsedText" title="">Image hidden</div>
    <div class="BBCodeCollapseControl"><svg fill="#FFFFFF" xmlns="http://www.w3.org/2000/svg" class="SVGIcon_Button SVGIcon_X" version="1.1" x="0px" y="0px" viewBox="-165 95 100 100">
        <g>
          <polygon points="-74.9,117.2 -102.2,145 -74.9,172.8 -89.1,186.8 -116.2,159.3 -143.2,186.8 -157.5,172.8 -130.2,145 -157.5,117.2 -143.2,103.2 -116.2,130.7 -89.1,103.2"></polygon>
        </g>
      </svg></div>
  </div>
</div>

**Comment by TheFabulousPika** [comment]
Collapsed YouTube video
```
<div class="msgText " data-copytext="">
  <div class="BBCodeResizableComponent BBCodeYouTubeComponent Minimized" data-copystyle="block" data-copytext="R.E.M. - Crush With Eyeliner (Official Music Video)
https://www.youtube.com/watch?v=ycvJHQUqU1M" style="width: 16px;">
    <div class="BBCodeCollapsed" title="R.E.M. - Crush With Eyeliner (Official Music Video)">Video hidden</div>
    <div class="BBCodeRestoreControl" title="Restore to original size">
      <div class="restoreSize"></div>
    </div>
    <div class="BBCodeResizeControl" title="Drag to resize video"><svg version="1.1" xmlns="http://www.w3.org/2000/svg" class="SVGIcon_Button SVGIcon_ResizeIcon" x="0px" y="0px" viewBox="0 0 256 256">
        <g>
          <path fill="#020202" d="M115.512,137.263l21.6-21.6c0,0,25.5,25.5,45.8,45.8l35-35v91.6h-91.6l35-35 C141.012,162.763,115.512,137.263,115.512,137.263z"></path>
          <path fill="#020202" d="M108.312,129.763c-20.3-20.3-35.8-35.8-35.8-35.8l21.6-21.601c0,0,15.5,15.5,35.8,35.801L108.312,129.763z"></path>
          <path fill="#020202" d="M54.312,75.763c-20.3-20.3-15.8-15.8-15.8-15.8l21.6-21.601c0,0-4.5-4.5,15.8,15.801L54.312,75.763z"></path>
        </g>
      </svg></div>
    <div class="BBCodeCollapseControl" title="Unhide video in chat"><svg fill="#FFFFFF" xmlns="http://www.w3.org/2000/svg" class="SVGIcon_Button SVGIcon_X" version="1.1" x="0px" y="0px" viewBox="-165 95 100 100">
        <g>
          <polygon points="-74.9,117.2 -102.2,145 -74.9,172.8 -89.1,186.8 -116.2,159.3 -143.2,186.8 -157.5,172.8 -130.2,145 -157.5,117.2 -143.2,103.2 -116.2,130.7 -89.1,103.2"></polygon>
        </g>
      </svg></div>
  </div>
</div>

**Comment by TheFabulousPika** [comment]
Fixed in #40
